man/fi_usnic.7.md

```diff
@@ -46,9 +43,26 @@ libfabric provider:
     MPI's *usnic* BTL.
   * *FI_EP_MSG* and *FI_EP_RDM* endpoints are implemented, but are
     only lightly tested.  It is likely that there are still some bugs
-    in these endpoint types.
-  * Other capabilities, such as the tag matching interface and RDMA
-    operations, are not yet implemented.
+    in these endpoint types.  RDMA is not yet supported.
+  * [`fi_provider`(7)](fi_provider.7.html) lists requirements for all
+    providers.  The following limitations exist in the *usnic*
+    provider:
+    * *fi_getname* is not supported on *FI_EP_DGRAM*, *FI_EP_MSG*, and
+      passive endpoints.
+    * multicast operations are not supported on *FI_EP_DGRAM* and
+      *FI_EP_RDM* endpoints.
+    * *FI_EP_MSG* endpoints only support connect, accept, and shutdown
+      operations.
+    * Passive endpoints only support listen operations.
```

Comment:
Might be best to say "CM operations" instead of "operations".  But that's minor.